Example/Podfile.lock

```diff
@@ -1,51 +1,63 @@
 PODS:
-  - Firebase/Core (3.14.0):
-    - FirebaseAnalytics (= 3.7.0)
-    - FirebaseCore (= 3.5.1)
-  - Firebase/RemoteConfig (3.14.0):
+  - Firebase/Core (4.7.0):
```

Comment:
why this huge version bump?

Reply:
because pod update.

Reply:
It's ok, it's just for the demo app. It's not a dependency in the podspec.

Reply:
LoL, so it is ok test the app with completely different lib version? this is a HUGE RED warning for me.